### PR TITLE
location bugfix

### PR DESCRIPTION
**Pull request by tristanhcole**
* unit tests
* functional tests

### CODE REVIEW

File: app/server/models/ip_address.py [hunk 1]
```diff
@@ -38,5 +38,5 @@ def ip(self, ip):
                 mt.set_ip_location(self.id, ip)
             except Exception as e:
                 print(e)
-                sentry_sdk.captureException(e)
+                sentry_sdk.capture_exception(e)
```
Comment (michielderoos) [inline app/server/models/ip_address.py]: My bad!

**Comment by tristanhcole** [comment]
WIP still!

**Comment by tristanhcole** [comment]
Ok, good for review @enjeyw @michielderoos !

File: app/server/api/user_api.py [hunk 2]
```diff
@@ -120,7 +120,7 @@ def get(self, user_id):
                     'users': user_list,
                 }
             }
-            return make_response(jsonify(response_object)), 201
+            return make_response(jsonify(response_object)), 200
```
Comment (tristanhcole) [inline app/server/api/user_api.py]: get should return 200

File: test/functional/app/api/test_user_api.py [hunk 3]
```diff
@@ -4,11 +4,65 @@
...
+        assert isinstance(data['user'], object)
+        assert data['user']['first_name'] == 'John'
+        assert data['user']['last_name'] == 'Smith'
+        assert data['user']['custom_attributes']['bio'] == 'EasyMart'
+        assert data['user']['custom_attributes']['gender'] == 'female'
+        assert data['user']['phone'] == proccess_phone_number(phone)
+        assert data['user']['is_vendor'] is False
+        assert data['user']['is_tokenagent'] is False
+        assert data['user']['is_groupaccount'] is False
+        assert data['user']['transfer_accounts'][0]['balance'] == 400
```
Comment (michielderoos) [inline test/functional/app/api/test_user_api.py]: This assertion is breaking in CircleCI

Comment (tristanhcole) [reply test/functional/app/api/test_user_api.py]: yeah it's werid because it passes locally ..

**Comment by tristanhcole** [comment]
[Location bug](https://trello.com/c/oLNhf00L/316-location-bug)

File: test/functional/app/api/test_user_api.py (hunk 3 above)
Comment (tristanhcole) [reply test/functional/app/api/test_user_api.py]: should be fixed :)

**Pull request merged**